server/src/main/java/com/cloud/vm/UserVmManagerImpl.java

```diff
@@ -4072,7 +4079,11 @@ public UserVmVO doInTransaction(TransactionStatus status) throws InsufficientCap
 
                 persistVMDeployAsIsProperties(vm, userVmOVFPropertiesMap);
 
-                _vmDao.saveDetails(vm);
+                List<String> hiddenDetails = new ArrayList<>();
```

Comment:
@shwstppr I think we can use a single line something like below to add display value.
        `super.addDetail(new UserVmDetailVO(resourceId, key, value, display))`

Reply:
@harikrishna-patnala using addDetail method directly here still won't make it single line as detail that will be added needs to be removed from vm.details,

```
if (customParameters.containsKey(VmDetailConstants.NAME_ON_HYPERVISOR)) {
    userVmDetailsDao.addDetail(vm.getId(), VmDetailConstants.NAME_ON_HYPERVISOR, customParameters.get(VmDetailConstants.NAME_ON_HYPERVISOR), false);
    customParameters.remove(VmDetailConstants.NAME_ON_HYPERVISOR);
}
```
Current code may allow setting display=false for multiple details in future